src/nvim/tui/tui.c

```diff
@@ -134,6 +136,7 @@ typedef struct {
 static bool volatile got_winch = false;
 static bool did_user_set_dimensions = false;
 static bool cursor_style_enabled = false;
+static bool tmux = false;
```

Comment:
normally we put this on `TUIData` struct, then access it in `tui_term_osc_cmd` via the `ui->data` pointer. Probably the others could have that treatment, but not worth changing since https://github.com/neovim/neovim/pull/10071 will change this stuff.

Reply:
This was marked as resolved?

Reply:
You said _not worth changing because https://github.com/neovim/neovim/pull/10071 will change this_?

Reply:
"Not worth changing" was referring to `did_user_set_dimensions` et al. Whereas `tmux` is being changed already in this PR, so it should not continue the pattern.